lib/engines/content_block_manager/app/services/content_block_manager/get_host_content_items.rb

```diff
@@ -4,18 +4,22 @@
 
 module ContentBlockManager
   class GetHostContentItems
-    attr_reader :content_id
+    attr_reader :content_id, :page, :order
 
-    def initialize(content_id:)
+    DEFAULT_ORDER = "-unique_pageviews".freeze
```

Comment:
does this mean it shows the pages with least page views first? Is this something we'll revisit, as I guess the more page views the more important the document is?

Reply:
This confused me too, but when we're doing ascending order with numbers, it shows the smallest first, so to see the most page views first we need descending order.